src/ProjectTask.php

```diff
@@ -1407,8 +1407,9 @@ public static function showFor($item)
                     'delete' => _x('button', 'Put in trashbin'),
                     'restore' => _x('button', 'Restore'),
                     'purge' => _x('button', 'Delete permanently')
-                ]
-            ]
+                ],
+                'action_button_classes' => 'btn btn-sm btn-outline-secondary me-2',
```

Comment:
Better to handle this directly in `datatable.html.twig` since most, if not all, sublists should be using this while regular Search lists are not.

Old:
```{% do call('Html::showMassiveActions', [massiveactionparams|default([])]) %}```
New:
```{% do call('Html::showMassiveActions', [{action_button_classes: 'btn btn-sm btn-outline-secondary me-2'}|merge(massiveactionparams|default({}))]) %}```

Reply:
fixed